### DISCUSSION TITLE
Version equivalency

### DISCUSSION THREAD

**Discussion by ucirello** [supermerill/SuperSlicer#3363] [General]
Hi,

Is there any version equivalency between SuperSlicer and PrusaSlicer? Is it correct to say the SuperSlicer 2.5 is the same as PrusaSlicer 2.5?

Thanks

**Comment by neophyl** [comment]
In most respects yes.  The various versions are built from PS of the same core version.   2.3.xx is equivalent to PS2.3.  2.4.xx is also equivalent to PS 2.4.  However you are not guaranteed to have all of the functions/features available from the PS version.  For example 2.5.xx does not currently have lightning infill but does have arachne.  
As far as I can tell 2.3.xx and 2.4.xx have everything PS2.3 and PS2.4 have,  and more, as SuSi has a lot more options.  However the 2.5 merge is only partial and a few things were added in the 2.5 test builds.
